pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py

```diff
@@ -456,18 +456,22 @@ def update_logger_connector(self, fx_name: str = None) -> None:
         logger_connector.callback_metrics.update(callback_metrics)
         logger_connector.callback_metrics.pop("epoch", None)
 
-    def run_batch_from_func_name(self, func_name) -> Mapping:
+    def run_batch_from_func_name(self, func_name) -> Dict:
         results = {}
         for fx_name, hook_result in self._internals.items():
             func = getattr(hook_result, func_name)
             results.update(func(latest=True, include_forked_originals=False))
         return results
 
-    def get_latest_batch_log_metrics(self) -> Mapping:
-        return self.run_batch_from_func_name("get_batch_log_metrics")
+    def get_latest_batch_log_metrics(self) -> Dict:
+        batch_log_metrics: Dict = self.run_batch_from_func_name("get_batch_log_metrics")
```

Comment:
just curious how this internal typing helps... ?

Reply:
Trying to resolve pyright.

Reply:
Removed